dist/index.html

```diff
@@ -4,31 +4,92 @@
     <meta charset="utf-8">
     <title>Seek-A-Design Playground</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7fcb54df80.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <script src="../src/index.js"></script>
   </head>
   <body>
     <div class="parent-container">
-      <!-- <h1 class="center">Hello World!</h1> -->
       <div class="navigation-container">
-        <button type="button" class="nav">Home</button>
-        <button type="button" class="nav">Projects</button>
-        <button type="button" class="nav">About Me</button>
-        <button type="button" class="nav">Contact</button>
+        <button type="button" class="nav-home">
```

Comment:
**Lines 14-24:**

There are many ways to accomplish this goal, and your way is one of them! 

However, *ideally*, a nav section of HTML will be nested inside a `<nav></nav>` tag. Then, each menu item will be a list item (`<li>`) in an unordered list (`<ul>`). Like this:

```
    <nav>
      <ul class="navbar">
        <li>
          HOME
        </li>
        <li>
          PROJECTS
        </li>
        ...
      </ul>
    </nav>
```

Our HTML is simple enough to make this refactor now, so let's make it happen! 

Removing the bullet points from the list might take a little CSS magic, but it looks like you've figured out that technique already in your `.seeking` class, so I'm confident you can accomplish this here as well. 👍

Reply:
Alright, I went ahead and reformatted this section of my code as directed. I removed the bullet points in an earlier task previously, so I was able to just copy and paste that css code. After changing <div> and <button> to <nav> <ul><li> elements,  will I still be able to use the same css that I had on my buttons? I went to Landons branch to check out his html/css so I could try transferring over the button styles over. Is there a way to view the files (as a webpage that I can inspect) on github, or do I need to download them and open them manually?

Reply:
This post is making me realize that we need to have a group call on GitHub, group project workflow, etc, to better enable the three of us to work as a team. Sorry if this has been difficult so far.

As far as the styling goes, for the most part, the CSS should be transferable, with only minor exceptions.